test/integration/test_expid.py

```diff
@@ -260,3 +279,337 @@ def test_copy_expid_no(autosubmit: Autosubmit) -> None:
     hpc_result = describe[4].lower()
 
     assert hpc_result != new_hpc
+
+
+def create_as_exp(tmp_path):
```

Comment:
Rename it to something like `_get_experiment_data`? Private function, and it returns the experiment data for `autosubmit_config` or `autosubmit_exp`.